documents/src/pages/elements/number-field.md

```diff
@@ -103,12 +103,12 @@ element.addEventListener('value-changed', (e) => {
 ```
 
 ## Input validation
-To validate input from users `ef-number-field` provides similar features to a native input. When a user assigns an invalid input to the control, it will automatically apply an error style to alert the user.
+To validate input from users `ef-number-field` provides similar features to a native input. When a user assigns an invalid input to the control, it will automatically apply an error style to alert the user. However, if you define a default value that is invalid you need to call the `input.reportValidity()` method during initialization to ensure the error style is applied.
```

Comment:
```suggestion
To validate input from users, `ef-number-field` provides similar features to a native input. When a user assigns an invalid input to the control, it will automatically apply an error style to alert the user. However, if you define a default value that is invalid, you need to call `input.reportValidity()` during initialization to ensure the error style is applied.
```